### ISSUE TITLE
Add custom CA to velero

### ISSUE THREAD

**Issue by blancharda** [defenseunicorns/uds-bundle-software-factory-nutanix#144]
Velero is currently unable to push contents to buckets. This may be related to custom certs / CAs.

Relates to #143 and #122

**Comment by jacobbmay** [comment]
I don't think this is specific to trusting certificates and is likely just an issue with the Nutanix objects API itself. Have hit issues with other tools using Nutanix objects (zarf registry, terraform before we updated to a version of tofu that added additional S3 flags that can be set). Terraform specifically was encountering the exact same error (I think it is related to the v4 streaming protocol) until we updated the version.

For reference, to get tofu to work with Nutanix objects over TLS we are setting the following options:
```
    skip_credentials_validation = true
    skip_region_validation      = true
    skip_requesting_account_id  = true
    skip_metadata_api_check     = true
    skip_s3_checksum            = true
```

**Comment by JoeHCQ1** [comment]
I doubt this will be worked since we need to remove Velero anyway.

**Comment by jacobbmay** [comment]
Velero isn't going anywhere. That is neuvector

**Comment by JoeHCQ1** [comment]
This may be fixed by an update a while ago, check to see if it's still an issue, if not, close, if yes, we may just keep running with http.
